packages/atomic/src/components/search/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -74,18 +74,18 @@ import {initializePopover} from '../atomic-popover/popover-type';
  * @part search-result-path - The search result path.
  * @part search-highlight - The highlighted query inside the facet values.
  *
- * @part parents - The parent values container.
- * @part all-categories-button - The "View all" button displayed first along the parents.
- * @part parent-button - The clickable parent button.
- * @part active-parent - The non-clickable active parent.
+ * @part parents - The container surrounding the whole hierarchy of values.
+ * @part sub-parents - The container surrounding a part of the hierarchy of values.
```

Comment:
'a part' sounds a big vague. Would the following make sense?
```suggestion
 * @part sub-parents - The second-level container, containing a part of the hierarchy of values.
```

Reply:
agreed, but it's not only second-level, technically there's as many `sub-parents` containers embedded in one another as there as the depth of the selected value.

Reply:
Ah that's interesting, I see why you wrote it this way then. Do you think the following would help:
```suggestion
 * @part sub-parents - The container surrounding a sub-hierarchy of values.
```